### PR TITLE
Feature/statistic-section

### PR DESCRIPTION
**Pull request by phongduong-dp**
# Description & Technical Solution

Create statistic section

# Checklist

- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] Already rebased against main branch.

### CODE REVIEW

File: src/components/layout/StatisticSection.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { AiOutlinePlus } from 'react-icons/ai';
+
+export type ContentType = {
+  title: string;
+  statistic: string;
+};
+
+type StatisticSectionType = {
+  content: ContentType[];
```
Comment (its-rav) [inline src/components/layout/StatisticSection.tsx:10]: put this in plural for clearer variable context. e.g. contentItems, statisticItems

File: src/components/layout/Section.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import clsxm from '@/lib/clsxm';
+
+export type VariantType = {
+  [key: string]: string[];
```
Comment (its-rav) [inline src/components/layout/Section.tsx:6]: use keyof VariantArr !? maybe

File: src/components/layout/Section.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+};
+
+type SectionType = {
```
Comment (its-rav) [inline src/components/layout/Section.tsx:9]: rename all of component props type to {componentName}Props => add a convention for all props type

File: src/components/layout/StatisticSection.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { AiOutlinePlus } from 'react-icons/ai';
+
+export type ContentType = {
+  title: string;
+  statistic: string;
```
Comment (its-rav) [inline src/components/layout/StatisticSection.tsx:6]: should this be number!?

**Review by its-rav — CHANGES_REQUESTED** [review]

**Comment by phongduong-dp** [comment]
@its-rav  Please review, Section.tsx has been delete because we have not used it yet. Only StatisticSection.tsx comments are picked up.

**Review by its-rav — APPROVED** [review]
lgtm

**Pull request merged**